packages/extension/src/ui/features/accountTokens/TokenIcon.tsx

```diff
@@ -45,7 +45,7 @@ export const TokenIcon: FC<TokenIconProps> = ({ name, url, size, verified, ...re
         />
       </Circle>
       {!verified && (
-        <Tooltip label={"Unverified Token"} shouldWrapChildren={true}>
+        <Tooltip label={"Unknown Token"} shouldWrapChildren={true}>
```

Comment:
Not sure if this is the proper label, but it's less misleading than `Unverified Token`.